worldeditadditions/utils/selector_helps.lua

```diff
@@ -18,5 +18,21 @@ function worldeditadditions.axis_left(axis,sign)
 	else return true, "x", -sign end
 end
 
+--- Dehumanize Direction: translates up, down, left, right, front, into xyz based on player orientation.
+-- @param	name	string	The name of the player to return facing direction of.
+-- @param	dir	string	Relative direction to translate.
+-- @return	Returns axis name and sign multiplier.
+function worldeditadditions.dh_dir(name, dir)
```

Comment:
I know talked about changing this name... I guess now is the time to decide what to call this.

Reply:
Renamed to `xlate_dir`.